etcdserver/snapshot_store.go

```diff
@@ -28,6 +30,12 @@ import (
 	dstorage "github.com/coreos/etcd/storage"
 )
 
+// Because raft state machine retries to send snapshot to slow follower when
+// receiving MsgHeartbeatResp from the follower, the newly created
+// snapshot should be gotten within one heartbeat interval.
+// Set it as 5s to match the upper limit of heartbeat interval.
+const closeUnusedSnapshotInterval = 5 * time.Second
```

Comment:
This is not the usually style to comment a const. First we usually say what this const is, then we say when to use it and why we set the value as it is.